demo-output.yaml

```diff
@@ -138,8 +138,76 @@
       - testing
       - test
       incidents:
+      - uri: file:///analyzer-lsp/cmd/analyzer/main.go
```

Comment:
These should not be added back, we got rid of these

Reply:
thats weird, gimme a sec

Reply:
i think something happened while rebasing

Reply:
wait no, that is the output that i am getting when i run the build. im not sure what the problem is